crates/ruff_linter/resources/test/fixtures/flake8_bandit/S603.py

```diff
@@ -1,20 +1,34 @@
 from subprocess import Popen, call, check_call, check_output, run
 
 # Different Popen wrappers are checked.
-Popen("true", shell=False)
-call("true", shell=False)
-check_call("true", shell=False)
-check_output("true", shell=False)
-run("true", shell=False)
+a = input()
+Popen(a, shell=False)
+call(a, shell=False)
+check_call(a, shell=False)
+check_output(a, shell=False)
+run(a, shell=False)
 
 # Values that falsey values are treated as false.
-Popen("true", shell=0)
-Popen("true", shell=[])
-Popen("true", shell={})
-Popen("true", shell=None)
+Popen(a, shell=0)
```

Comment:
Hmm, it seems it was very deliberate to flag string literals. I wonder why

Reply:
Okay, upstream seems to agree that string literals should be safe: https://github.com/PyCQA/bandit/issues/333#issuecomment-404233717